tutorminio/plugin.py

```diff
@@ -5,7 +5,7 @@
 from glob import glob
 
 import importlib_resources
-from tutor import hooks as tutor_hooks
+from tutor import hooks as tutor_hooks, plugins
```

Comment:
We cannot import the "plugins" module from external plugins, as the plugins.py API is not considered stable. Instead, to guess whether the discovery plugin is enabled, we should check whether the "discovery" string is present in the PLUGINS configuration setting.

Something like: (just a suggestion)

```
...
    "DISCOVERY_BUCKET_NAME": "{% if PLUGINS.get('discovery') %}discoveryuploads{% endif %}",
...
{% if MINIO_DISCOVERY_BUCKET_NAME %}
mc mb --ignore-existing minio/{{ MINIO_DISCOVERY_BUCKET_NAME }}
...
{% endif %}

Reply:
@regisb, can u have a look at it again, Thanks.

Reply:
Sorry I missed that notification. I have an intuition that your implementation of `LOADED_PLUGINS` is going to cause us lots of issues down the road. That's because it is highly dependent on the order in which plugins are loaded. Did you try my suggested implementation above? If yes, what didn't work?

Reply:
Actually i was trying to import it.
But it's working great. Thanks.